lib/sisimai/bite/email/exim.rb

```diff
@@ -330,7 +330,7 @@ def scan(mhead, mbody)
             recipients = rcptinhead.size
 
             rcptinhead.each do |e|
-              # Insert each recipient address into @$dscontents
+              # Insert each recipient address into dscontents
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->